src/app/static/src/app/components/modelCalibrate/ModelCalibrate.vue

```diff
@@ -4,72 +4,102 @@
             <loading-spinner size="lg"></loading-spinner>
             <h2 id="loading-message" v-translate="'loadingOptions'"></h2>
         </div>
-        <dynamic-form ref="form"
```

Comment:
Sorry, I think a lot of these changes were made by my auto-formatting so this page is a little noisy.